go/pkg/db/queries/key_list_live_by_api_id.sql

```diff
@@ -0,0 +1,95 @@
+-- name: ListLiveKeysByApiID :many
+SELECT
+    k.*,
+    sqlc.embed(a),
+    sqlc.embed(ka),
+    sqlc.embed(ws),
+    i.id as identity_table_id,
```

Comment:
just name it identity_id

Reply:
I tried and when I did this it automatically marks this as `IDENTITY_ID_2` and pretty much breaks everything.

Reply:
ah yeah, thats because you select k.* which has its own identity id column

Reply:
but we can't rely on that because we need to join the identities table and check if the identity id is correct and if its not deleted.

Reply:
Yeah kept it as is and filters and joins are working fine atm

Reply:
yeah good catch lemme add that check too

Reply:
`LEFT JOIN identities i ON k.identity_id = i.id AND i.deleted = false` nope there it is 😄

Reply:
yeah I noticied it too after writing the comment 😅